models/project_board.go

```diff
@@ -32,17 +33,19 @@ const (
 
 // ProjectBoard is used to represent boards on a project
 type ProjectBoard struct {
-	ID      int64 `xorm:"pk autoincr"`
-	Title   string
-	Default bool `xorm:"NOT NULL DEFAULT false"` // issues not assigned to a specific board will be assigned to this board
+	ID       int64 `xorm:"pk autoincr"`
+	Title    string
+	Default  bool `xorm:"NOT NULL DEFAULT false"` // issues not assigned to a specific board will be assigned to this board
+	Priority int
```

Comment:
I think it should be not null with default value, probably 0?

```suggestion
	Priority int `xorm:"NOT NULL DEFAULT 0"`
```